migration-helper/modelmesh-to-raw.sh

```
@@ -0,0 +1,2025 @@
+list_and_select_inference_services
+
+# Cache available templates early to avoid repeated API calls
+cache_available_templates
+# Create serving runtimes for migration
+create_serving_runtimes
+
+# Process the models for migration, prepare the InferenceService manifests
+process_inference_services
+
+# Clean up any empty directories that may have been created
+cleanup_empty_directories
```

Shellcheck (0.11.0)

[error] This function is only defined later. Move the definition up.

(SC2218)